src/shared/components/adminTabs/Games/GamesManagement.tsx

```diff
@@ -53,11 +53,40 @@ function GamesManagement() {
         })])
     }
 
+    function areDistinct(arr)
+    {
+        let n = arr.length;
+    
+        let s = new Set();
+        for (let i = 0; i < n; i++) {
+            s.add(arr[i]);
+        }
+
+        return (s.size == arr.length);
```

Comment:
Let's use `===` instead of `==`, because we may get a `true` if the `arr.length` is undefined